samples/subsys/fs/fat_fs/boards/mg100.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_SPI=y
+CONFIG_DISK_DRIVER_SDMMC=y
```

Comment:
There is not benefit for the project to have this configuration file overlay, especially as there is nothing specific to MG100 board and it can be set during build either directly or via ``west``.

Reply:
We should not have board configuration overlays like this. You can update documentation for your board to improve user knowledge so that user know how to enable this functionality also for his own application if you really bother about experience. You could add common configuration overlay. Btw,  DISK_DRIVER_SDMMC is `default y if DT_HAS_ZEPHYR_SDMMC_DISK_ENABLED`, CONFIG_SPI=y is sufficient, what is just fine to pass during build command.